apps/mini-rx-angular-demo/src/app/modules/todos/components/todos-shell/todos-shell.component.html

```diff
@@ -41,13 +42,13 @@
                         <span>Todos Done</span>
                     </div>
                     <app-todo-list
-                        [todos]="(todosDone$ | async)!"
-                        [selectedTodo]="(selectedTodo$ | async)!"
+                        [todos]="viewModel.todosDone"
+                        [selectedTodo]="viewModel.selectedTodo"
                         (selectTodo)="todosState.selectTodo($event)"
                     ></app-todo-list>
                 </div>
             </div>
-            <div class="col" *ngIf="selectedTodo$ | async as selectedTodo">
+            <div class="col" *ngIf="viewModel.selectedTodo as selectedTodo">
```

Comment:
Do you still need to cast objec type?

Reply:
Do you mean the "ngIf as" syntax?

It works also without Observables.

The alternative would be:

![image](https://github.com/spierala/mini-rx-store/assets/1272446/fce42c5f-342b-4a89-a06b-f5b619f6d825)

Which one do you prefer?

Reply:
Both should be fine

Reply:
Removed the ngIf as syntax.

It has indeed no benefit here.